config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -168,6 +168,7 @@ presubmits:
         - |
           set -euo pipefail
           make gofail-enable
+          export JUNIT_REPORT_DIR=${ARTIFACTS}
```

Comment:
Curious on how this artifact dir is intended to work? Is there a bucket these artifacts will be stored in after the job has completed?

Reply:
They show up in the "Artifacts" tab of a build. For example, clicking "Artifacts" on [build 1800476731895713792](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-etcd-unit-test-amd64/1800476731895713792) takes you to the [bucket](https://gcsweb.k8s.io/gcs/kubernetes-jenkins/pr-logs/pull/etcd-io_etcd/18158/pull-etcd-unit-test-amd64/1800476731895713792/). Inside there's an "artifacts" directory with the JUnit XML results.

I'm porting the following fragment, which seems to have been overlooked in the initial migration: https://github.com/etcd-io/etcd/blob/57907746e39da97ed3ce2fbd1f97e8ceb2ce8199/.github/workflows/tests-template.yaml#L69-L73.

However, I'm open to dropping saving the artifacts if we don't find value in storing them 🙂.

Reply:
Thanks for the explanation (and for updating it into etcd-io/etcd docs with screenshot) this is great functionality lets continue to save these.

Reply:
To be clear and 100% transparent, the screenshots you saw were from the Robustness tests. These are JUnit results :smile: